contrib/IECoreArnold/src/IECoreArnold/ParameterAlgo.cpp

```diff
@@ -370,6 +370,9 @@ int parameterType( IECore::TypeId dataType, bool &array )
 		case BoolDataTypeId :
 			array = false;
 			return AI_TYPE_BOOLEAN;
+		case V3fDataTypeId :
+			array = false;
+			return AI_TYPE_VECTOR;
 		case M44fDataTypeId :
```

Comment:
Does this case imply M44f would work in the new test case as well? Someone was asking if it could, so I guess I can say it already does?